packages/mermaid/src/docs.mts

```diff
@@ -41,6 +41,8 @@ import { remark } from 'remark';
 // @ts-ignore No typescript declaration file
 import flatmap from 'unist-util-flatmap';
 
+const version = JSON.parse(readFileSync('package.json', 'utf8')).version;
```

Comment:
Wondering if we should use this version or the latest version from npm/jsdelivr?

Reply:
Should we just hardcode the version to:
`https://cdn.jsdelivr.net/npm/mermaid@9/dist/mermaid.min.js`?

Right now it redirects to v9.1.7, and it should automatically redirect to the latest `^9.0.0` version on jsdelivr.

Reply:
Done